cmd/bootstrap-peer-server.go

```diff
@@ -107,7 +107,7 @@ func getServerSystemCfg() ServerSystemConfig {
 		if _, ok := skipEnvs[envK]; ok {
 			continue
 		}
-		envValues[envK] = env.Get(envK, "")
+		envValues[envK] = logger.HashString(env.Get(envK, ""))
```

Comment:
cherry-picked not properly, please refer to https://github.com/minio/minio/pull/16853/files#diff-3618d106abc87ba433cbc51da0aacf1427b4dbfde9d2385083fb1c47e54f20d9